src/App.tsx

```diff
@@ -20,12 +30,16 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={(event) => (
+                  setQuery(event.target.value)
```

Comment:
You can create a function and past her name. This will make the code easier to read